apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/[keyId]/page.tsx

```
@@ -155,6 +160,41 @@ export default async function APIKeyDetailPage(props: {
+            name: permission.name,
+            description: permission.description,
+            checked: role.permissions.some((p) => p.permissionId === permission.id),
+            part: p,
+            permissions: {},
+            path: parts.slice(0, i).join("."),
+          };
+        }
+        n = n[p].permissions;
+      }
+    }
+    let data = {
```

Biome

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)